hathor/p2p/states/ready.py

```diff
@@ -68,7 +71,16 @@ def __init__(self, protocol: 'HathorProtocol') -> None:
         })
 
         # Initialize sync manager and add its commands to the list of available commands.
-        self.sync_manager = NodeSyncTimestamp(self.protocol, reactor=self.reactor)
+        self.sync_manager: SyncManager
+        if protocol.protocol_version is ProtocolVersion.V1:
+            self.log.debug('loading sync-v1')
+            self.sync_manager = NodeSyncTimestamp(self.protocol, reactor=self.reactor)
+        elif protocol.protocol_version is ProtocolVersion.V2:
+            self.log.debug('loading sync-v2 (pretend)')
+            # self.sync_manager = NodeBlockSync(self.protocol, reactor=self.reactor)
```

Comment:
Can we remove this old code?

Reply:
I can remove this commented line, but this is actually new code that will be "uncommented" on the next PR.